src/api/flaskr/service/shifu/admin.py

```
@@ -179,6 +223,70 @@ def _normalize_metadata_json(value: Any) -> Dict[str, Any]:
     return {}
 
 
+def _normalize_credit_amount(value: Any) -> Decimal:
+    normalized = str(value or "").strip()
+    if not normalized:
+        raise_param_error("amount")
+    try:
+        parsed = _quantize_credit_amount(Decimal(normalized))
+    except Exception:  # pragma: no cover - Decimal error details vary
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)